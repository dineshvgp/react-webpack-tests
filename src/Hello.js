import React from 'react'
import Loadable from 'react-loadable'

const Header = Loadable({
  loader: () => import('./Header'),
  loading: () => null
})

function Hello () {
  return (
    <div>
      <Header />
      <h1>Hello from React </h1>
    </div>
  )
}

export default Hello
