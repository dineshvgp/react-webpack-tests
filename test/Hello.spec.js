import React from 'react'
import Hello from '../src/Hello'

it('should render a label', () => {
    const wrapper = mount(
      <Hello />
    );
    expect(wrapper).toMatchSnapshot();
});
