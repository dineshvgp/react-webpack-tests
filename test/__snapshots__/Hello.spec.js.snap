// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a label 1`] = `
<Hello>
  <div>
    <LoadableComponent>
      <loading
        error={null}
        isLoading={true}
        pastDelay={false}
        timedOut={false}
      />
    </LoadableComponent>
    <h1>
      Hello from React 
    </h1>
  </div>
</Hello>
`;
